sqlite/migrations/0006_migrate_replications_foreign_key.up.sql

```diff
@@ -0,0 +1,34 @@
+PRAGMA foreign_keys=off;
```

Comment:
I'm not sure that this is strictly necessary, though I was going off of [this reference](https://www.techonthenet.com/sqlite/foreign_keys/drop.php#:~:text=How%20to%20Drop%20a%20Foreign%20Key%20on%20a%20Table,data%20into%20the%20new%20table) for how to drop the foreign key in a migration, which included the pragma

Reply:
I'd like a better explanation of why we would or wouldn't want this before this merges.

Reply:
Hey @lesam, from the sqlite docs:
```
If foreign key constraints are enabled when it is prepared,
the DROP TABLE command performs an implicit DELETE to
remove all rows from the table before dropping it. The
implicit DELETE does not cause any SQL triggers to fire,
but may invoke foreign key actions or constraint violations.

...

The properties of the DROP TABLE and ALTER TABLE commands
described above only apply if foreign keys are enabled.
If the user considers them undesirable, then the
workaround is to use PRAGMA foreign_keys to disable
foreign key constraints before executing the DROP or
ALTER TABLE command.
```

If we didn't turn off the foreign keys, then the DROP of the old table would still trigger the cascading delete (the exact behavior we don't want). While having the pragma off allows us to potentially violate foreign key constraints, since we're doing a simple copy in a non-multi-connection environment, I believe it is safe.

https://www.sqlite.org/foreignkeys.html